content/chainguard/libraries/faq.md

```diff
@@ -12,6 +12,71 @@ weight: 003
 toc: true
 ---
 
+## What security issues can Chainguard Libraries prevent?
+
+As detailed in the [background](/chainguard/libraries/overview/#background) and
+[introduction](/chainguard/libraries/overview/#introduction) Chainguard
+Libraries are built directly from source in the Chainguard Factory and the
+resulting binaries are directly provided to you by Chainguard. Chainguard
+operates the whole supply chain for the package lifecycle as one reliable,
+secure partner. You can therefore avoid issues from the following software
+supply chain attacks surface points:
+
+* Build pipeline
+* Build system
+* Dependency injection
+* Bypass of CI/CD systems
+* Library distribution
+* Library consumption
+
+More information about the stages is available on the [Supply chain Levels for
+Software Artifacts (SLSA) website](https://slsa.dev/).
+
+Following are examples that exploited issues in the relevant supply
+chain:
+
+**Example: Ultralytics Python project**
+
+* Attackers compromised Ultralytics' GitHub Actions workflow, injecting malware
+  into PyPI package releases.
+* Malicious versions proliferated (8.3.41, 8.3.42, 8.3.45, 8.3.46).
+* Affected a widely-used AI library with ~60 million downloads included a cryptominer.
+* No source code was included and Chainguard Libraries would therefore block.
```

Comment:
I think this needs to be spelled out more for people less familiar with the product. I am not sure if I phrased it very elegantly here, so feel free to iterate

```suggestion
* No source code was included: the use of Chainguard Libraries, based on source code, would have prevented the attack.
```

Reply:
Fixed